Microsoft.NET.Build.Containers/AuthHandshakeMessageHandler.cs

```diff
@@ -179,4 +188,9 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
 
     [GeneratedRegex("(?<key>\\w+)=\"(?<value>[^\"]*)\"(?:,|$)")]
     private static partial Regex BearerParameterSplitter();
+
+    private static string GetCacheKey(Uri uri)
```

Comment:
Duplicated code here

Reply:
It doesn't look like either this method or the GetCacheKey inside of AuthHeaderCache is called. Do we need this?

Reply:
Yeah, I should have moved it but instead copied it. The copy in `AuthHeaderCache` is called in the other methods there.